packages/expo-updates/android/src/main/java/expo/modules/updates/db/entity/UpdateEntity.kt

```diff
@@ -60,5 +61,11 @@ class UpdateEntity(
   @ColumnInfo(name = "failed_launch_count", defaultValue = "0")
```

Comment:
I think the fields can be in the constructor?

```
  @field:ColumnInfo(name = "manifest") var manifest: JSONObject,
  @field:ColumnInfo(name = "url") var url: Uri?,
  @field:ColumnInfo(name = "headers") var requestHeaders: Map<String, String>?
```

Doing so exposes a good place we should add explicit null arguments: EmbeddedUpdate. That way it's clear that they are in the null case in selection policies.